.gitignore

```diff
@@ -34,3 +34,5 @@ cypress/cucumber-json
 
 # IDE
 .idea
+
+src/components/Navigation/reacticons.md
```

Comment:
What's about this md-file, a react-icons cheat sheet 😀

Reply:
We put our react icons that we wanted to use there later on since we didn't quite understand how to use them with styled-components yet 👍